Content.Shared/SimpleStation14/Prototypes/JukeboxTrackPrototype.cs

```diff
@@ -0,0 +1,73 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Audio;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.SimpleStation14.Prototypes;
+
+[Prototype("jukeboxTrack")] [Serializable] [NetSerializable]
+public sealed class JukeboxTrackPrototype : IPrototype
+{
+    [IdDataField]
+    public string ID { get; } = default!;
+
+    /// <summary>
+    ///     Input string for the name, pre-localization.
+    /// </summary>
+    [DataField("name")]
+    private string NameString { get; } = default!;
+
+    /// <summary>
+    ///     Actual name, localized.
+    /// </summary>
+    public string Name => Loc.GetString(NameString);
+
+    /// <summary>
+    ///     Path of this track's audio file.
+    /// </summary>
+    [DataField("path")]
+    public SoundSpecifier Path { get; } = default!;
+
+    /// <summary>
+    ///     Serialized as a string because TimeSpan is not serializable, and converted to a TimeSpan with <see cref="ToTimeSpan"/>.
+    /// </summary>
+    [DataField("duration")]
+    private string DurationString { get; } = default!;
+
+    /// <summary>
+    ///     Actual duration as a TimeSpan.
+    /// </summary>
+    public TimeSpan Duration => ToTimeSpan(DurationString);
```

Comment:
We should just learn how to make a serializer... I honestly don't think it'd be *that* hard.